src/code/z_play.c

```diff
@@ -1237,10 +1237,10 @@ void Play_Draw(PlayState* this) {
 
                     this->pauseBgPreRender.fbuf = gfxCtx->curFrameBuffer;
                     this->pauseBgPreRender.fbufSave = (u16*)gZBuffer;
-                    func_800C1F20(&this->pauseBgPreRender, &sp70);
+                    PreRender_SaveFramebuffer(&this->pauseBgPreRender, &sp70);
```

Comment:
More of a side note but at some point we really need to decide if we're using "framebuffer" or "frame buffer", and use it consistently.. (iirc it was supposed to be the latter)

Reply:
Yeah definitely, it's worthy of note that libultra VI functions use framebuffer/Framebuffer so if we want to be 100% consistent we would have to use the same